selfdrive/ui/spinner/spinner.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
```

Comment:
The spinner is launched while openpilot native components are being compiled. It'll be moved to NEOS in the next NEOS release.